src/app/busca-ativa/gestantes/Gestantes.tsx

```
@@ -28,7 +35,7 @@ export const Gestantes : React.FC<GestantesProps> = ({
     const router = useRouter();
     const path = usePathname();
     const visao = session?.user.perfis.includes(5) || session?.user.perfis.includes(8) ? "aps" : "equipe"
-    useEffect(() => {router.push(`${path}?visao=${visao}`)}, [visao]);
+    useEffect(() => {router.push(`${path}?aba=${activeTitleTabIndex}&sub_aba=${activeTabIndex}&visao=${visao}`)}, [visao, activeTabIndex, activeTitleTabIndex]);
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: path

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)